infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -188,7 +191,15 @@ public ServerContext(Map<String, Object> serverConfig) {
         // Metrics setup & reporting configuration
         if (!isMetricsReportingSetUp(metrics)) {
             MetricsUtils.metricsReportingSetup(metrics);
+            logMetricsSize();
         }
+
+        FileChannelPerf.setupParameters(OP_SPIKE_CNT, OP_SPIKE_LATENCY, false);
+        logMetricsSize ();
```

Comment:
Why we execute `logMetricsSize()` two times? We execute this method inside an `if` condition several lines above and a second time in here